heron/spi/tests/java/com/twitter/heron/spi/packing/PackingPlanTest.java

```diff
@@ -25,7 +25,7 @@
 import com.twitter.heron.spi.common.Constants;
 
 public class PackingPlanTest {
-  public static PackingPlan generatePacking(Map<String, List<String>> basePacking) {
+  private static PackingPlan generatePacking(Map<String, List<String>> basePacking) {
     Resource resource =
```

Comment:
Access isn't needed outside of this class and won't be when I add a serde test, since I was going to add it to this class as well.